commcare_connect/static/css/tom-select.css

```diff
@@ -0,0 +1,429 @@
+/**
```

Comment:
does this not come with the package somewhere? it is weird we have to commit their whole css